changelog.asc

```diff
@@ -2,6 +2,45 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.19.0 (20241025)
+* Incorporating the work from three Gsoc projects and the work done in the last couple of months, we are proud to present the latest iteration of the best Android app for learning!  
+* We really appreciate the https://opencollective.com/ankidroid[donations], they paid for these features 🤝
+* Features
+** Anki upstream version 24.06.3 included
+** Two new widgets for easier access to your decks and cards
+** Streamlined process of adding cloze notes to AnkiDroid through an Instant Add Note Editor
+** Improved the process of adding multimedia to your notes
+** Improved experience for devices with larger screens
+*** Better utilization of screen space
+*** Better mouse support
+*** Better keyboard shortcuts support
+*** Drag and drop support on some parts of the application
+** Better support for cards flags management
+** SHA256 & SHA1 application certificate fingerprints can be viewed in repository README
+** Update audio focus with support for Android 15
+** Reduced application size
+** Improved accessibility
+*** Better support for larger text size
+*** Added info text for more UI elements
+** Allow native deck selection in statistics screen
+** Study options screen shows bury counts as desktop
+** App shows info about WebView version requirements at start
```

Comment:
I think this is only relevant to developers since we are trying to reproduce on emulators that don't receive WebView updates, everyone else gets updates over the air

```suggestion
```